stac_fastapi/pgstac/models/links.py

```diff
@@ -129,7 +129,8 @@ def link_next(self) -> Optional[Dict[str, Any]]:
         if self.next is not None:
             method = self.request.method
             if method == "GET":
-                href = merge_params(self.url, {"token": f"next:{self.next}"})
+                url = self.resolve(f"collections/{self.collection_id}/items")
```

Comment:
😬 it seems that `PagingLinks` class is not only used in the `/items` endpoint but also in the `all_collections` method

Reply:
oh I missed that. I guess that change is not going to work then